free-1.md

```diff
@@ -56,13 +155,33 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+I am not entirely sure how to implement languages quite yet, so I don't think I
+have an accurate judgment of this at the moment. I do, however, think that this
+may require a lot of systems code to work in the context of an editor. At this
+point I would still think language design would be a major part of the project,
+so my guess would be 50-70% of time would be spent on language design. 
 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
-
+I think this is a pretty ambitious project. While I think the impact is relatively
```

Comment:
I like this idea overall. Creating graphs isn't limited to economics, so you can reach a wider group of people who want to create simple graphs on a computer.